### PR TITLE
M3 issue5 paco

### PR DESCRIPTION
**Pull request by fraramfra**
- Added a variable random_seed to split_and_fit.py
- Included it in the command.bash set to 123 (as it had been ran before to give same results)
- Included the new command in README file

Ran script multiple times and it always yields the same result from my computer:
"The model obtained a final test score of: 0.7976336816226183"

Is this what you had in mind?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,7 +77,8 @@ python scripts/eda.py \
 python scripts/split_and_fit.py \
     --processed_dir="data/processed/cleaned_data.csv" \
     --results_dir="results/figures" \
-    --preprocessed_dir="data/processed"
+    --preprocessed_dir="data/processed" \
+    --random_seed=123
```
Comment (mikem2m) [inline README.md:81]: Nitpicky but can you change this to `522`? Otherwise its good :)

Comment (mikem2m) [reply README.md:81]: @fraramfra

**Review by mikem2m — CHANGES_REQUESTED** [review]
Seed 123 -> 522

File: command.bash [hunk 2]
```diff
@@ -13,7 +13,8 @@ python scripts/eda.py \
 python scripts/split_and_fit.py \
     --processed_dir="data/processed/cleaned_data.csv" \
     --results_dir="results/figures" \
-    --preprocessed_dir="data/processed"
+    --preprocessed_dir="data/processed" \
+    --random_seed=123
```
Comment (mikem2m) [inline command.bash:17]: And here

**Comment by fraramfra** [comment]
I've pushed the updates to my branch. How do I update this PR? Or is the PR automatically for your review?

**Comment by mikem2m** [comment]
> I've pushed the updates to my branch. How do I update this PR? Or is the PR automatically for your review?

Should be automatic @fraramfra

**Review by mikem2m — APPROVED** [review]
LGTM!

**Pull request merged**